Malaria_analysis.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p>Nice job! I liked how you did some data exploration to see the column names and possible values before writing the filter statement. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>